spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -85,7 +85,11 @@ def get_value(self, name):
             return self.call_kernel(
                 interrupt=True, blocking=True).get_value(name)
         except (TimeoutError, UnpicklingError):
-            raise ValueError('Kernel did not send the value.')
+            msg = _("The variable is either too big or not picklable."
+                    "<br><br>"
+                    "<i>Note</i>: Please don't report this problem on "
+                    "Github, there's nothing to do about it.")
```

Comment:
I understand the idea, but this is a weird message to give to the user :-|

Reply:
There's really nothing we can do about it.